tests/MongoDB.Analyzer.Tests.Common.TestCases/EF/NotSupportedEFExpressions.cs

```diff
@@ -21,10 +21,10 @@ namespace MongoDB.Analyzer.Tests.Common.TestCases.EF;
 
 public sealed class NotSupportedEFExpressions : TestCasesBase
 {
-    [NotSupportedEF("Byte array type is not supported by this version of the EF provider.", DriverVersions.Linq3OrGreater)]
-    [NotSupportedEF("Byte array type is not supported by this version of the EF provider.", DriverVersions.Linq3OrGreater)]
-    [NotSupportedEF("Byte array type is not supported by this version of the EF provider.", DriverVersions.Linq3OrGreater)]
-    [NotSupportedEF("Byte array type is not supported by this version of the EF provider.", DriverVersions.Linq3OrGreater)]
+    [NotSupportedEF("Byte array type is not supported by this version of the EF provider.")]
```

Comment:
Oh, I added support for byte arrays a while back in our EF provider. Can probably be a separate PR to add support.

Reply:
Nice, added [VS-167](https://jira.mongodb.org/browse/VS-167) to track that.